2-mandatory/2-financial-times.js

```diff
@@ -14,68 +24,108 @@ function potentialHeadlines(allArticleTitles) {
     (you can assume words will always be seperated by a space)
 */
 function titleWithFewestWords(allArticleTitles) {
-    // TODO
+  // TODO
+
+  headlines = potentialHeadlines(allArticleTitles);
+
+  let shortHeadline = [];
+  let fewestWords = 100;
+
+  for (article of allArticleTitles) {
+    articleSplitter = article.split(" ");
+
+    if (articleSplitter.length < fewestWords) {
+      fewestWords = articleSplitter.length;
+      shortHeadline = article;
+    }
+  }
+
+  return shortHeadline;
 }
 
 /*
     The editor of the FT has realised that headlines which have numbers in them get more clicks!
     Implement the function below to return a new array containing all the headlines which contain a number.
     (Hint: remember that you can also loop through the characters of a string if you need to)
 */
+
 function headlinesWithNumbers(allArticleTitles) {
-    // TODO
+  // TODO
+
+  let headlinesWithNumbers = [];
+
+  for (singleTitle of allArticleTitles) {
+    let numberChecker = /[0-9]/.test(singleTitle);
+
+    if (numberChecker === true) {
```

Comment:
Normally when you have an `if` statement that look like `if (numberChecker === true) {`, you can instead write `if (numberChecker) {`.
Can you think of why that works?